SQL/Archive/2018-04-06_Change_no_nii_option_to_create_nii.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE ConfigSettings 
+    SET Name="create_nii", 
+        Description="Create NIfTI files if set to 1", 
```

Comment:
@cmadjar 

just a small comment; this should read 'if set to Yes' instead of '1' because the front end changes the booleans to become Yes/No.
This is not applicable only to your change here but all other settings as well; but I thought I would comment FYI

(redmine #13200)

Reply:
@MounaSafiHarab really good catch! I made the changes but just realized it got merged so I'll create a separate PR with those changes :). Thanks for bringing that up!!

Reply:
Oh, I see the ticket is assigned to you @MounaSafiHarab. Let me know if you prefer doing the changes and I'll test it or if you prefer that I do the changes and you test it. I'm open to both solutions. Just let me know.

Reply:
lol see when it was assigned! a while ago (right after the release that brought those changes to the front end!), I never got to do it; so if you want please take the ticket (and if you don't; I do not mind at all doing it when I get the chance).

Reply:
haha. OK. I'll do it while this is fresh in my head then ;)

Reply:
Done! See PR #3577 :)